source/CommonJobs/CommonJobs.Application.EvalForm/Commands/GetEvaluationCalifications.cs

```diff
@@ -141,11 +141,7 @@ private CalificationsDto GetFinishedEvaluation(CalificationsEvaluationDto evalua
         {
             return new CalificationsDto()
             {
-                View = (_loggedUser == evaluationDto.ResponsibleId || _isEmployeeManager)
-                ? UserView.Company
-                    : (_loggedUser == _evaluatedUser)
-                        ? UserView.Auto
-                        : UserView.Evaluation,
+                View = UserView.Company,
```

Comment:
As I understand it, `UserView` represents what kind of user is seeing the evaluation, in my opinion, if behavior should not be modified based on the kind of user, it should not take into account `UserView` value.

Also, as we see, there are a lot of references to `UserView` value, I am not sure if it is safe to change it without check all of them.